glue-etl/src/main/scala/com/circle/data/utils/QueryBase.scala

```diff
@@ -69,8 +71,10 @@ object QueryBase {
   ): DataFrame = {
     val selectCols = Array(
       "profile.si_userprofile_id AS `userId`",
-      "action.name AS `actionName`",
-      "activity.*" 
+      "action.name AS `event`",
+      "activity.event_object",
+      "activity.event_details",
+      """date_format(to_timestamp(activity.created_at), "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'") AS `timestamp`"""
```

Comment:
Does this mean we could have 'actions' that occurred before the timestamp of the corresponding identify for a particular user? If I remember correctly you wanted to set timestamp to now (for identify)

Reply:
Good call. I will use the user created_at as timestamp for identify events. That change will be a different PR.